index.html

```diff
@@ -30,43 +30,68 @@
         <a href="about.html" onclick="closeNav()">About</a>
       </div>
     </div>
-    <nav class="nav-container navbar-background" id="nav-icon">
-      <nav class="navbar navbar-light">
+    <nav class="nav-container navbar-background p-0" id="nav-icon">
+      <nav class="navbar navbar-light mobile-only">
         <button class="navbar-toggler col-2" onclick="openNav()">
           <span class="navbar-toggler-icon"></span>
         </button>
       </nav>
-      <nav class="menu-items">
-      <a href="#pro-cards" onclick="closeNav()">Main Program</a>
-      <a href="#feat-cards" onclick="closeNav()">Featured Speakers</a>
-      <a href="about.html" onclick="closeNav()">About</a>
-      </nav>
+      <div class="w-100 d-nav align-items-center">
+        <img class=" col-3 mx-auto logo menu-items align-self-center pb-2" src="src/ailogo.png" alt="">
+        <nav class="menu-items col-8 align-content-center m-0 h-100 me-5">
+          <a href="#pro-cards" onclick="closeNav()">Main Program</a>
+          <a href="#feat-cards" onclick="closeNav()">Featured Speakers</a>
+          <a href="about.html" onclick="closeNav()">About</a>
+          <button class="h-50 align-self-center secondary head-btn">AI
+            Campaign</button>
+        </nav>
+      </div>
+
     </nav>
   </header>
   <!-- Header -->
-  <section class="header-container header-background p-3 w-100">
-    <h3>"Hello! Entrepreneurs"</h3>
-      <h1>Angel Investor Business Summit 2022</h1>
-      <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Doloribus, iure officiis? Eveniet eos nulla, unde
-        voluptatum officiis animi rerum! Rem, quos illum ratione incidunt reiciendis obcaecati ad ipsa eius quisquam?
-      </p>
-      <h2>2022.12.25(SUN) ~ 27(TUE)</h2>
-      <h6>@ Jamaica Conference Center, Conference Room 1</h6>
+  <section class="header-container header-background p-3 w-100 pt-md-5 ps-md-5 pe-md-5 pb-md-5">
+    <h3 class="mt-md-5 ms-md-5 ps-md-5 me-md-5 pe-md-5">"Hello! Entrepreneurs"</h3>
+    <h1 class="ms-md-5 ps-md-5 me-md-5 pe-md-5">Angel Investor Business Summit 2022</h1>
+    <p class="ms-md-5 ps-md-5 me-md-5 pe-md-5 pt-md-5 pb-md-5">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Doloribus, iure officiis? Eveniet eos nulla, unde
+      voluptatum officiis animi rerum! Rem, quos illum ratione incidunt reiciendis obcaecati ad ipsa eius quisquam?
+    </p>
+    <h2 class="ms-md-5 ps-md-5">2022.12.25(SUN) ~ 27(TUE)</h2>
+    <h6 class="ms-md-5 ps-md-5">@ Jamaica Conference Center, Conference Room 1</h6>
   </section>
   <!-- Main program -->
```

Comment:
- [ ]  Follow the Provided Behance design fonts, color, and margin.  Please resemble the provided design 

Your design 
![image](https://user-images.githubusercontent.com/43877689/181496103-00e822ce-757b-40f0-bc1e-bda4f6749050.png)

Behance design 
![image](https://user-images.githubusercontent.com/43877689/181496435-08ca87b2-e61f-4515-afba-fb2269d07aef.png)